lib/event_summarizer/idv_matcher.rb

```diff
@@ -333,7 +333,7 @@ def handle_ipp_enrollment_status_update(event:)
       verified = tmx_status != 'review' && tmx_status != 'reject'
 
       if verified
-        current_idv_attempt.event << SignificantIdvEvent.new(
+        current_idv_attempt.significant_events << SignificantIdvEvent.new(
```

Comment:
I am not positive this is correct, but I know `.event` was wrong.

Reply:
per this comment: https://gsa-tts.slack.com/archives/C03KDV9S8T1/p1745596588542599?thread_ts=1745592464.360409&cid=C03KDV9S8T1 this is correct.